packages/api/src/cache/clients/RedisClient.ts

```diff
@@ -36,7 +36,7 @@ export default class RedisClient extends BaseClient {
       (err: Error) => this.logger?.error(err) || console.error(err),
     )
 
-    return this.client.connect()
+    await this.client.connect()
```

Comment:
@cannikin Just retroactively confirming I haven't done something stupid here and that this a fine change?

Reply:
Huh...so previously the connect() was sync and now it's not? Or should this have always had an `await` here?

Think it's okay to drop the `return`? Does it hurt to keep it in there along with the `await`?

Reply:
The return had to go because the function returns void. Then I still have to await the call so that it's settled within the function. The connect function was already async. It no longer returns a promise but settles it internally - since it's not allowed to return it.